ufl/finiteelement/elementlist.py

```diff
@@ -259,6 +259,9 @@ def show_elements():
 register_element("Gauss-Lobatto-Legendre Edge L2", "GLL-Edge L2", 0, L2, "L2 Piola", (0, None),
                  ("interval",))
 
+# directly-defined serendipity elements ala Arbogast
+register_element2("Sdirect", 0, H1, "physical", (1, None), ("quadrilateral",))
```

Comment:
Should this be rather defined on `cubes`, not just quad?

Reply:
Todd tells me there is an extension to hexes, but they have to have straight faces (which is not true unless the four vertices of a face are coplanar), and I don't know if it is written up.  I have only implemented quads.

Reply:
We plan working on some quadrilateral H(div) families, but the implementation is not finished yet.  Should we register them now so we don't need a separate future request to ufl?